Runtime error when there are no NVM devices.

There is a runtime error when cAdvisor is built with `libipmctl` and there are no NVM devices.

```
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/google/cadvisor/nvm.getAvgPowerBudget(0x0, 0x0, 0x0) 
        /go/src/github.com/google/cadvisor/nvm/machine_libipmctl.go:63 +0x587
github.com/google/cadvisor/nvm.GetInfo(0x0, 0x0, 0x0, 0x0, 0x0) 
        /go/src/github.com/google/cadvisor/nvm/machine_libipmctl.go:110 +0x18d
github.com/google/cadvisor/machine.Info(0x1755780, 0x20f8158, 0x1748120, 0xc000446ff0, 0x1705601, 0xc000447050, 0x0, 0x1748120) 
        /go/src/github.com/google/cadvisor/machine/info.go:82 +0x1b2
github.com/google/cadvisor/manager.New(0xc000559780, 0x1755780, 0x20f8158, 0xc000236668, 0xc000236674, 0xc00040da70, 0xc00040dad0, 0xc00035a580, 0x1, 0x1, ...)
        /go/src/github.com/google/cadvisor/manager/manager.go:208 +0x4b6
main.main()
        /go/src/github.com/google/cadvisor/cmd/cadvisor.go:173 +0x306
```

--- a/nvm/machine_libipmctl.go
+++ b/nvm/machine_libipmctl.go
@@ -57,6 +57,11 @@ func getAvgPowerBudget() (uint, error) {
 		return uint(0), fmt.Errorf("Unable to get number of NVM devices. Status code: %d", err)
 	}
 
+	if count == 0 {
+		klog.Warningf("There are no NVM devices!")
+		return uint(0), nil
+	}
+
 	// Load basic device information for all the devices
 	// to obtain UID of the first one.
 	devices := make([]C.struct_device_discovery, count)

